txtl-liposome_water-in-oil.md

```diff
@@ -0,0 +1,217 @@
+inside of a lipid-based container. The liposomes created by these
+protocols are between 1 and 10 um in diameter and should be usable
+with an cell-free protein expression system.
+
+The preparation of liposomes using w/o emulsions as template requires
+four steps:
+
+### **Step 0**: Deposition of a thin lipid film in a glass vial
+
+#### Materials:
+
+* Glass vials (any will do, but we use [2mL Fisherbrand Class B Clear
+  Glass Threaded vials: cat #:
+  03-339-21A](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738)
+
+* POPC in chloroform: [https://avantilipids.com/product/850457](https://avantilipids.com/product/850457) 
+
+* Lyss-Rhod-PE:
+  [https://avantilipids.com/product/810150](https://avantilipids.com/product/810150)
+
+* Cholesterol in chloroform (25 mg/ml): https://avantilipids.com/product/700000
+
+* Glass syringes of various sizes (10 uL, 50 uL, 250 uL, and 1 mL are
+  suggested ex. Hamilton gastight cat #: 14-815-238, but any will do)
+  for use with lipids and chloroform
+  [https://www.hamiltoncompany.com/products/syringes-and-needles/general-syringes/gastight-syringes](https://www.hamiltoncompany.com/products/syringes-and-needles/general-syringes/gastight-syringes)
+
+### **Step 1**: Preparation of the oil containing lipids
+
+#### Materials:
+
+* Mineral oil: [https://us.vwr.com/store/product/7422728/mineral-oil-light-white-high-purity-grade](https://us.vwr.com/store/product/7422728/mineral-oil-light-white-high-purity-grade)
+
+### **Step 2**: Self-assembly of the Liposomes by centrifugation
+
+#### Materials:
+
+* 20 uM of HPTS stock solution: ([https://www.thermofisher.com/order/catalog/product/H348](https://www.thermofisher.com/order/catalog/product/H348))
+
+* 100 mM HEPES, 200 mM glucose pH 8
+
+* 2M Sucrose Stock
+
+* 100 mM HEPES + 250 mM glucose, pH 8
+
+* 100 mM HEPES, 200 mM sucrose pH 8
+
+### **Step 3**: Microscopy Visualization
+
+#### Materials:
+
+* Frame Seals: [https://www.thermofisher.com/order/catalog/product/S24736](https://www.thermofisher.com/order/catalog/product/S24736)
+
+* Glass slides
+
+## Step 0: Deposition of a thin lipid film in a glass vials
+
+The first step in the protocol is to generate the lipid film required
+to to form the lipid-in-mineral oil solution. Makes 6 samples with
+accurate measurement. Each vial will have ~ 15 mg of lipid with 0.1
+mol % of 18:1 Lyss-Rho-PE.
+
+1. Create lipid master mix in a glass beaker 
+
+    a. POPC/Lyss-Rhod-PE 
+
+        i. Add  4 mL (100 mg) of POPC in chloroform (25 mg/ml) 
+
+        ii. Add 200 uL (0.2 mg) of Lyss-Rho-PE in chloroform (1 mg/mL) 
+
+        iii. Swirl gently until all POPC is dissolved and color is homogeneous.
+
+    b. POPC/Cholesterol/Lyss-Rhod-PE 
+
+        i. Add 2.668 mL (66.7 mg) of POPC in chloroform (25 mg/ml) 
```

Comment:
This type of list is not valid MarkDown and so it gets formatted as a code block.  We should see if there is a way to fix.  Possible number the steps as b.1, b.2, etc?